src/main/scala/org/globalforestwatch/summarystats/SummaryMain.scala

```diff
@@ -161,7 +169,9 @@ object SummaryMain
           tclOpt,
           gladOpt,
           changeOnlyOpt,
-          sensitivityTypeOpt
+          fireAlertTypeOpt,
+          fireAlertSourceOpt
+//          sensitivityTypeOpt
```

Comment:
We can discuss the fireAlertSource. But maybe I should just refactor how we're taking in input parameters so we're not always having to comment out parameters. Maybe a config case class instead of an N-tuple?